stories/vertical-navigation/vertical-navigation.stories.js

```diff
@@ -0,0 +1,252 @@
+export default {
+    title: 'Components/Vertical Navigation',
+    parameters: {
+        description: `
+The Vertical Navigation can be used as an alternative to the Horizontal Navigation.<br>
+This navigation type is composed by two distinct item levels:<br>
+
+- First Level items - always visible, can be either navigable in and of themselves or be a container for Second Level items (it should not be both)
```

Comment:
Rephrase syntax: 

First Level items →  Always visible, can be either navigable in and of themselves or be a container for Second Level items (it should not be both).

Second Level items → Are always contained by a First Level item, and can have their visibility toggled by this First Level item. They are always only navigable.